app/components/MainSidebar.tsx

```diff
@@ -53,159 +54,84 @@ interface MainSidebarProps {
         variants={{
           open: { width: "100%" },
           closed: { width: 80 },
         }}
-        className="h-full bg-base-100 w-full max-w-xs flex shadow-lg z-10 overflow-hidden"
+        className="h-full bg-base-100 w-full max-w-xs flex flex-col shadow-lg z-10"
       >
-        <MaxHeightContainer
-          header={
-            <div className="p-2">
-              <Popover
-                isOpen={isPopupOpen}
-                positions={["right"]}
-                padding={16}
-                onClickOutside={() => setIsPopupOpen(false)}
-                content={
-                  <div className="mt-2">
-                    <BandSelectPopup
-                      bands={bands}
-                      onSelect={() => setIsPopupOpen(false)}
-                      onShowNew={() => {
-                        setIsPopupOpen(false);
-                        setShowCreateMenu(true);
-                      }}
-                    />
-                  </div>
-                }
+        <div className="p-2">
+          <Popover
+            isOpen={isPopupOpen}
+            positions={["right"]}
+            padding={16}
+            onClickOutside={() => setIsPopupOpen(false)}
+            content={
+              <div className="mt-2">
+                <BandSelectPopup
+                  bands={bands}
+                  onSelect={() => setIsPopupOpen(false)}
+                  onShowNew={() => {
+                    setIsPopupOpen(false);
+                    setShowCreateMenu(true);
+                  }}
+                />
+              </div>
+            }
+          >
+            <button
+              className="btn btn-block btn-outline h-auto p-2 hover:btn-accent"
+              onClick={() => setIsPopupOpen(!isPopupOpen)}
+            >
+              <BandOption
+                band={band}
+                isCollapsed={openState === 'closed'}
+                memberRole={memberRole}
               >
-                <button
-                  className="btn btn-block btn-outline h-auto p-2 hover:btn-accent"
-                  onClick={() => setIsPopupOpen(!isPopupOpen)}
-                >
-                  <BandOption
-                    band={band}
-                    isCollapsed={!isOpen}
-                    memberRole={memberRole}
-                  >
-                    <FontAwesomeIcon icon={faSort} />
-                  </BandOption>
-                </button>
-              </Popover>
+                <FontAwesomeIcon icon={faSort} />
+              </BandOption>
+            </button>
+          </Popover>
+        </div>
+
+        <div className="flex flex-col h-full p-2 gap-4">
+          <fetcher.Form method="post" action="/resource/user-settings" className="relative h-4">
+            <div className="absolute -right-6 z-20">
+              <Button type="submit" name="sidebar-state" value={openState === 'open' ? 'closed' : 'open'} size="sm" isRounded kind="active" ariaLabel="Toggle sidebar">
```

Comment:
name here is the name of the key within the cookie, value is what it gets set to